lib/lifecycle.js

```diff
@@ -134,14 +134,17 @@ async function runBenchmark(bench, initialIterations, repeatSuite, minSamples) {
 
 	const opsSec = totalIterations / (totalTimeSpent / timer.scale);
 
+	const sampleData = histogram.samples;
+
 	return {
 		opsSec,
 		iterations: totalIterations,
 		// StatisticalHistogram is not a serializable object
 		histogram: {
-			samples: histogram.samples.length,
+			samples: sampleData.length,
 			min: histogram.min,
 			max: histogram.max,
+			all: sampleData,
```

Comment:
```suggestion
			sampleData,
```

I'd name it as sampleData instead